src/lib/cdev/CMakeLists.txt

```diff
@@ -35,17 +35,21 @@ set(SRCS_PLATFORM)
 if (${PX4_PLATFORM} STREQUAL "nuttx")
 	list(APPEND SRCS_PLATFORM
 		nuttx/cdev_platform.cpp
+		nuttx/cdev_platform.hpp
 	)
 else()
 	list(APPEND SRCS_PLATFORM
 		posix/cdev_platform.cpp
+		posix/cdev_platform.hpp
```

Comment:
Why is this required?

Reply:
It helps IDEs that directly support CMake (VSCode, CLion, etc) resolve everything and it makes the header dependency explicit. It makes no difference for the actual build.